src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/layout.html.twig

```diff
@@ -92,7 +92,7 @@
                     {% set forward_profile = profile.childByToken(request_collector.forwardtoken) %}
                     {% set controller = forward_profile ? forward_profile.collector('request').controller : 'n/a' %}
                     <div class="status status-compact">
-                        <span class="icon icon-bottom">{{ include('@WebProfiler/Icon/arrow-corner-down-right.svg') }}</span>
+                        <span class="icon icon-bottom">{{ include('@WebProfiler/Icon/forward.svg') }}</span>
```

Comment:
```suggestion
                        <span class="icon icon-bottom">{{ source('@WebProfiler/Icon/forward.svg') }}</span>
```

Quoting the doc block:

> Returns a template content without rendering it.

So it's faster to use `source` than `include` when the file is not a Twig file.